packages/webcert/src/page/ListPage.tsx

```diff
@@ -105,10 +106,17 @@ const ListPage: React.FC<Props> = ({ type, excludePageSpecificElements }) => {
       return noDraftsImage
     }
   }
+  console.log(listError)
 
   const getList = () => {
-    if (error) {
-      return <InfoBox type="error">Sökningen kunde inte utföras.</InfoBox>
+    if (listError) {
+      return (
+        <InfoBox type="error">
+          <div>
```

Comment:
Is this div necessary?

Reply:
I could look into moving it into the InfoBox component, would probably be better there. Without this div, we get an undesired layout since InfoBox only expects one child element.

Reply:
Aha, could <></> work? :)

Reply:
Nope, that will still create multiple children for the HTML element, so the CSS would still have the same problem.

Reply:
Ok then :)